transformer/kubernetes/kubernetestransformer.go

```diff
@@ -46,8 +47,9 @@ type Kubernetes struct {
 
 // KubernetesYamlConfig stores the k8s related information
 type KubernetesYamlConfig struct {
-	IngressName string `yaml:"ingressName"`
-	OutputPath  string `yaml:"outputPath"`
+	IngressName                string `yaml:"ingressName"`
+	OutputPath                 string `yaml:"outputPath"`
+	UseDefaultValuesInK8sYamls bool   `yaml:"useDefaultValuesInK8sYamls"`
```

Comment:
Would it be better to use the same name in all configs? like say `setDefaultValuesInYamls`?

Reply:
Yes sure

Reply:
Done.